enforcer_cached.go

```diff
@@ -47,6 +49,11 @@ func (e *CachedEnforcer) EnableCache(enableCache bool) {
 	e.enableCache = enableCache
 }
 
+// AutoClear determines whether to clear cache after any operation for policy. if AutoClear is true, cache will be clear after update policy.
+func (e *CachedEnforcer) AutoClearCache(autoClearCache bool) {
```

Comment:
not necessary

Reply:
I think there may be users who are used to the way. Without this, will some users be troubled?

Reply:
when policy changes, we must clear cache, so `AutoClearCache`  is unnecessary.

Reply:
ok